tests/test_openclhpp.cpp

```diff
@@ -4517,4 +4517,38 @@ void testgetObjectInfo() {
     TEST_ASSERT_EQUAL(type, CL_GL_OBJECT_BUFFER);
     TEST_ASSERT_EQUAL(bufobj, 0);
 }
+static cl_int clEnqueueReleaseGLObjects_testenqueueReleaseGLObjects(
+    cl_command_queue command_queue, cl_uint num_objects,
+    const cl_mem *mem_objects, cl_uint num_events_in_wait_list,
+    const cl_event *event_wait_list, cl_event *event, int num_calls) {
+    (void)command_queue;
+    TEST_ASSERT_EQUAL(1, num_objects);
+    TEST_ASSERT_NOT_NULL(mem_objects);
+    TEST_ASSERT_EQUAL(1, num_events_in_wait_list);
+    TEST_ASSERT_NOT_NULL(event_wait_list);
+    TEST_ASSERT_EQUAL(0, num_calls);
+    if (event != nullptr) {
+            *event = make_event(1);
+    }
+    return CL_SUCCESS;
+}
+
+void testenqueueReleaseGLObjects() {
+    cl::Memory obj;
+    cl::vector<cl::Memory> mem_objects = {obj};
```

Comment:
`mem_objects` (and hence `obj`) are input values to the function we are testing, so we need to ensure they are passed through correctly.  We should be able to use an object from the bufferPool for it.